Makefile

```diff
@@ -1,9 +1,12 @@
 # Makefile for pvc-autoresizer
 
-K8S_VERSION = 1.18.9
-KUBEBUILDER_VERSION = 2.3.1
-CTRLTOOLS_VERSION = 0.5.0
-KUSTOMIZE_VERSION = 3.7.0
+ENVTEST_K8S_VERSION = 1.19.2
+KUBEBUILDER_VERSION = 3.1.0
+CTRLTOOLS_VERSION = 0.4.1
```

Comment:
we use v0.5.0 instead of kubebuilder default version. Would you set this to previous version?

Reply:
The diff of controller-tools from v0.4.1 to v0.5.0:
https://github.com/kubernetes-sigs/controller-tools/compare/v0.4.1...v0.5.0

- https://github.com/kubernetes-sigs/controller-tools/pull/522
- https://github.com/kubernetes-sigs/controller-tools/pull/528
- https://github.com/kubernetes-sigs/controller-tools/pull/534
- https://github.com/kubernetes-sigs/controller-tools/pull/538
- https://github.com/kubernetes-sigs/controller-tools/pull/511

Reply:
The diff of controller-tools from v0.5.0 to v0.6.0:
https://github.com/kubernetes-sigs/controller-tools/compare/v0.5.0...v0.6.0

- https://github.com/kubernetes-sigs/controller-tools/pull/557
- https://github.com/kubernetes-sigs/controller-tools/pull/563
- https://github.com/kubernetes-sigs/controller-tools/pull/565
- https://github.com/kubernetes-sigs/controller-tools/pull/541
- https://github.com/kubernetes-sigs/controller-tools/pull/564
- https://github.com/kubernetes-sigs/controller-tools/pull/576

Reply:
Sorry @toshipp, If fixed version.
https://github.com/topolvm/pvc-autoresizer/pull/41/commits/bedef6f2642954f727b17fe1397bf63c30aee164

BTW, We can upgrade controller-tools to v0.6.0 too.
Should we upgrade controller-tools to v0.6.0?
Details of upgrade is here: https://github.com/topolvm/pvc-autoresizer/pull/41#discussion_r654900539

I think this upgrade doesn't have any effect on pvc-autoresizer probably.

Reply:
For TopoLVM, we update tools as well as libraries, but for pvc-autoresizer, we have no specific update policies.
I also think the update is safe, I welcome to accept the upate.

Reply:
Thank you. I updated controller-tools to v0.6.0.
https://github.com/topolvm/pvc-autoresizer/pull/41/commits/17a6935b02036fecd1f4bf1cabc0748ceecf89ca